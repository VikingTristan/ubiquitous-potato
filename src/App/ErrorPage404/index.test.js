import React from "react";
import { shallow } from "enzyme";

import ErrorPage404 from "./index";

describe("ErrorPage404: index", () => {
    it("is defined", () => {
        expect(ErrorPage404).toBeDefined();
    });

    it("renders", () => {
        const wrapper = shallow(<ErrorPage404 />);

        expect(wrapper).toMatchSnapshot();
    });
});
