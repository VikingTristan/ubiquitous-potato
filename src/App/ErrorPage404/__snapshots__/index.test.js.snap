// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage404: index renders 1`] = `
<Fragment>
  <h1>
    This is not the page you are looking for.
  </h1>
</Fragment>
`;
